pyproject.toml

```diff
@@ -1,3 +1,61 @@
+[project]
+name = "launchable"
+authors = [
+    {name = "Launchable, Inc.", email = "info@launchableinc.com"}
+]
+description = "Launchable CLI"
+readme = "README.md"
+license = {text = "Apache Software License v2"}
+requires-python = ">=3.13"
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "License :: OSI Approved :: Apache Software License",
+    "Operating System :: OS Independent",
+]
+dependencies = [
+    "click>=8.1",
+    "requests>=2.25",
+    "urllib3>=1.26",
+    "junitparser>=2.0.0",
```

Comment:
in main branch, we updated the junitparser to v4

Reply:
Okay, I'm going to backport the main branch. Thank you for noticing!